src/scenario/scenario_events_import_xml.c

```diff
@@ -90,7 +95,14 @@ static const xml_parser_element xml_elements[XML_TOTAL_ELEMENTS] = {
     { "request_immediately_start", xml_import_create_action, 0, "actions" },
     { "show_custom_message", xml_import_create_action, 0, "actions" },
     { "tax_rate_set", xml_import_create_action, 0, "actions" },
-    { "building_count_any", xml_import_create_condition, 0, "conditions" },
+    { "building_count_any", xml_import_create_condition, 0, "conditions" }, //40
+    { "custom_variable_check", xml_import_create_condition, 0, "conditions" },
+    { "trade_route_open", xml_import_create_condition, 0, "conditions" },
+    { "trade_route_price", xml_import_create_condition, 0, "conditions" },
+    { "change_custom_variable", xml_import_create_action, 0, "actions" },
+    { "change_trade_route_open_price", xml_import_create_action, 0, "actions" },
+    { "variables", xml_import_start_custom_variables, 0, "events" },
+    { "custom_variable", xml_import_create_custom_variable, 0, "variables" },
```

Comment:
Why not just `variable`? The `custom_variable` tag seems too verbose...

Reply:
Just here for the XML, or do you mean throughout the PR code?

Reply:
Just the xml, the rest of the code is okay.

Reply:
XML naming updated.